modules/registry-configuring-storage-baremetal.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+//* architecture/installation-.adoc
+
+[id="registry-configuring-storage-baremetal_{context}"]
+= Configuring registry storage for bare metal
+
+As a cluster administrator, following installation you must configure your
+registry to use storage.
+
+.Prerequisites
+
+* Cluster administrator permissions.
+* A cluster on bare metal.
+* PV with `ReadWriteMany` access mode, such as Azure File or NFS.
```

Comment:
```suggestion
* A provisioned Persistent Volume with `ReadWriteMany` access mode, such as Azure File or NFS.
```
You may also want out-link to upstream docs on PVs: https://v1-13.docs.kubernetes.io/docs/concepts/storage/persistent-volumes/